tests/vmi_lifecycle_test.go

```diff
@@ -1213,55 +1213,6 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			)
 		})
 
-		Context("VirtualMachineInstance Emulation Mode", decorators.SoftwareEmulation, func() {
-
-			It("[test_id:1644]should be reflected in domain XML", func() {
-				vmi := libvmifact.NewAlpine()
-				err := kubevirt.Client().RestClient().Post().Resource("virtualmachineinstances").Namespace(testsuite.GetTestNamespace(vmi)).Body(vmi).Do(context.Background()).Error()
-				Expect(err).ToNot(HaveOccurred(), "Should post the VMI")
-
-				listOptions := metav1.ListOptions{}
-
-				Eventually(func() int {
-					podList, err := kubevirt.Client().CoreV1().Pods(testsuite.GetTestNamespace(vmi)).List(context.Background(), listOptions)
-					Expect(err).ToNot(HaveOccurred(), "Should list the pods")
-					return len(podList.Items)
-				}, 75, 0.5).Should(Equal(1), "There should be only one pod")
-
-				Eventually(func() error {
-					podList, err := kubevirt.Client().CoreV1().Pods(testsuite.GetTestNamespace(vmi)).List(context.Background(), listOptions)
-					Expect(err).ToNot(HaveOccurred(), "Should list the pods")
-					for _, item := range podList.Items {
-						if strings.HasPrefix(item.Name, vmi.ObjectMeta.GenerateName) {
-							return nil
-						}
-					}
-					return fmt.Errorf("Associated pod for VirtualMachineInstance '%s' not found", vmi.Name)
-				}, 75, 0.5).Should(Succeed(), "Should find the VMI pod")
-
-				getOptions := metav1.GetOptions{}
-				var newVMI *v1.VirtualMachineInstance
-
-				newVMI, err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(newVMI)).Get(context.Background(), vmi.Name, getOptions)
-				Expect(err).ToNot(HaveOccurred(), "Should get VMI")
-
-				domain := &api.Domain{}
-				context := &converter.ConverterContext{
-					VirtualMachine: newVMI,
-					AllowEmulation: true,
-				}
-				err = converter.Convert_v1_VirtualMachineInstance_To_api_Domain(newVMI, domain, context)
-				Expect(err).ToNot(HaveOccurred())
-
-				expectedType := ""
-				if _, err := os.Stat("/dev/kvm"); errors.Is(err, os.ErrNotExist) {
```

Comment:
Following this change, the standard `errors` and `os` packages are no longer needed.